### ISSUE TITLE
Standard parts exported in wrong size

### ISSUE THREAD

**Issue by pschmutz** [Rhoban/onshape-to-robot#127]
Using this example document https://cad.onshape.com/documents/82f6d830dd1d95c3f57f9bc3/w/9d326290f62aa21685913619/e/63495dc78a6cb9f6c9011616

With the following config

    {
        "documentId": "82f6d830dd1d95c3f57f9bc3",
        "outputFormat": "urdf",
    }

The model output looks like this
![Screenshot from 2024-10-05 12-33-51](https://github.com/user-attachments/assets/b26ff485-d056-41ea-b09a-74c5698f1f0a)

While the original model looks like this

![image](https://github.com/user-attachments/assets/ca606b6a-e160-4864-8873-f3582e9b90ae)

The file names are correct though

![image](https://github.com/user-attachments/assets/c2648644-1ddd-48e6-aaf3-2dfb900155e2)

**Comment by Gregwar** [comment]
Hello, just tried this on the latest release and it appear to work better
Can you confirm it's ok for you ?

![Image](https://github.com/user-attachments/assets/d7020d83-98cf-427f-a934-e8746c9ff40c)

**Comment by pschmutz** [comment]
Yes, the screw size is correct now. Thank you so much for fixing this and maintaining the tool. It is really useful :)
